tide_core.install

```diff
@@ -1548,3 +1548,36 @@ function tide_core_update_8064() {
     }
   }
 }
+
+/**
+ * Add contact field to user account(business name, phone, notes).
+ */
+function tide_core_update_8065() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [\Drupal::service('extension.list.module')->getPath('tide_core') . '/config/install'];
+  $configs = [
+    'field.storage.user.field_business_contact_number' => 'field_storage_config',
+    'field.storage.user.field_business_name' => 'field_storage_config',
+    'field.storage.user.field_notes' => 'field_storage_config',
```

Comment:
So you are creating the field storage here, I don't see anything to create the fields though. e.g - `field.field.user.user.field_business_contact_number.yml`
`field.field.user.user.field_business_name.yml`
`field.storage.user.field_business_contact_number.yml`